binaries/conda/build_packages.sh

```diff
@@ -4,12 +4,18 @@ set -eou pipefail
 
 PYTHON_VERSIONS="3.6 3.7 3.8"
 PKGS="torchserve torch-model-archiver"
+TORCHSERVE_VERSION=`cat ../../ts/version.txt`
```

Comment:
For our shell scripts we prefer to use the `$()` notation versus back-ticks for executing commands.

Just to keep things consistent.

Reply:
Done.